pom.xml

```diff
@@ -304,7 +304,7 @@
         <version.com.google.guava>30.1-jre</version.com.google.guava>
         <version.com.google.guava.failureaccess>1.0.1</version.com.google.guava.failureaccess>
         <version.com.google.j2objc>1.3</version.com.google.j2objc>
-        <version.com.google.protobuf>3.15.7</version.com.google.protobuf>
+        <version.com.google.protobuf>3.17.3</version.com.google.protobuf>
```

Comment:
@jasondlee Could you have a look at 3.18.2 or 3.19.2 ? Both include a fix for CVE-2021-22569 https://github.com/protocolbuffers/protobuf/security/advisories/GHSA-wrvw-hg22-4m67 CVSS Score: 7.5